timetracker/entries/migrations/0003_auto_20150727_0752.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entries', '0002_auto_20150723_0819'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Client',
+            fields=[
+                ('id', models.AutoField(auto_created=True, serialize=False, primary_key=True, verbose_name='ID')),
+                ('name', models.CharField(max_length=200)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='project',
+            name='client',
+            field=models.ForeignKey(to='entries.Project'),
```

Comment:
When you make a mistake like this, you should delete the migration file and fix things, otherwise your migrations reach a broken state...